src/ndt7.d.ts

```diff
@@ -0,0 +1,81 @@
+declare module '@m-lab/ndt7' {
+    type URLType = '///ndt/v7/download' | '///ndt/v7/upload';
+    type URLs = Record<URLType, string>;
+
+    interface Location {
+        city: string;
+        country: string;
+    }
+
+    interface ServerChoice {
+        machine: string;
+        location: Location;
+        urls: URLs;
+    }
+
+    interface Config {
+        userAcceptedDataPolicy?: boolean;
+        mlabDataPolicyInapplicable?: boolean;
+    }
+
+    interface DiscoverServerURLsConfig extends Config {
+        loadbalancer?: string;
+        protocol?: string;
+        server?: string;
+    }
+
+    interface DownloadTestConfig extends Config {
+        downloadworkerfile?: string;
+    }
+
+    interface UploadTestConfig extends Config {
+        uploadworkerfile?: string;
+    }
+
+    interface Callbacks {
+        error?: (error: string) => void;
+    }
+
+    interface DiscoverServerURLsCallbacks extends Callbacks {
+        serverDiscovery?: (options: { loadbalancer: string | URL }) => void;
```

Comment:
should I change this to `loadbalancerURL` to avoid confusion with `loadbalancer` in `DiscoverServerURLsConfig`?